src/common/ui-components/Slider/Slider.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Slider from 'react-slick';
+import classNames from 'classnames/bind';
+
+import { Feature } from '../../../domain/api/generated/types.d';
+import Slide from '../Slide/Slide';
+import styles from './Slider.module.scss';
+
+const cx = classNames.bind(styles);
+
+export interface SimpleSliderProps {
+  readonly features: Feature[];
```

Comment:
>features: Feature[];

Instead of features as props we would like to have a way to select sets of categories and tags by clicking. As in the designs these slides are for example "Veneilijöiden palvelut kartalla" which would link to categories like marinas, cafes of islands and so on.

Reply:
Not sure what this means, we could bring this discussion to slack maybe?